.github/workflows/ci.yaml

```diff
@@ -40,3 +40,7 @@ jobs:
 
       - name: Run unit tests
         run: npm run test:unit
+
+      - name: Run integration tests
+        run: npm run test:integration
+
```

Comment:
I think the integration tests in vscode never have worked. When I said that we should re-enable them, I thought we were talking about LS? Anyhoo, if you get them running, no harm done :D

Reply:
I did say `snyk-ls` in my message on Slack but meant `vscode-extension` 🤦 But I added some new tests to VSCode that I would like to run in the pipeline so this is still useful if there's no harm done